portfolio/src/main/webapp/index.html

```diff
@@ -26,60 +27,63 @@ <h1 id="main-heading">Kira Toal's Portfolio</h1>
+          Additionally, I like to be active, so I love playing water polo, softball, or simply
+          going for a hike with my dogs.">
+            Hobbies
+          </li>
+          <li data-message="I absolutely adore horror movies (some of my favorites include 
+          Hereditary, It Follows, The Exorcist, The Shining, and Midsommar), horror video games, 
+          horror manga and books … pretty much anything related to my favorite genre! I also enjoy 
+          learning languages, and am currently studying Japanese in university and learning some 
+          Russian and ASL from my mother in my spare time.">
+            Interests
+          </li>
+          <li id="secret-text">???</li> 
+        </ul>
       </nav>
     </div>
     <div id="extra-content">
-			<section id="movie-recommendation">
-				<p id="recommend" onclick="recommendMovie()"> Recommend</p>
-				<p id="filler-text"> me a </p>
-				<select id="subgenres">
-					<option value="spn" id="supernatural" onclick="recommendMovie()">supernatural horror movie</option>
-					<option value="non-spn" id="non-supernatural" onclick="recommendMovie()">non-supernatural horror movie</option>
-				</select>
-				<div id="result-parent">
-					<p id="result"> <i>result ...</i> </p>
-				</div> 
-			</section>
-			<div id="creepy-images">
-			<a href="https://www.viz.com/junji-ito" target="_blank"><img class="ito-images" src="images/Ito1.jpg" /></a>
-			<a href="https://store.steampowered.com/app/913740/WORLD_OF_HORROR/" target="_blank"><img class="ito-images" src="images/Ito2.jpg"/></a>
-			<a href="http://www.craigdavidson.net/nickcutter/index.html" target="_blank"><img class="ito-images" src="images/Ito3.jpg"/></a>
-			</div>
+    <section id="movie-recommendation">
+      <p id="recommend" onclick="recommendMovie()"> Recommend</p>
+      <p id="filler-text"> me a </p>
+      <select id="subgenres">
+        <option value="spn" id="supernatural" onclick="recommendMovie()">supernatural horror movie</option>
+        <option value="non-spn" id="non-supernatural" onclick="recommendMovie()">non-supernatural horror movie</option>
+      </select>
+      <div id="result-parent">
+	<p id="result"> <i>result ...</i> </p>
+      </div> 
+    </section>
+    <div id="creepy-images">
+    <a href="https://www.viz.com/junji-ito" target="_blank"><img class="ito-images" src="images/Ito1.jpg" /></a>
+    <a href="https://store.steampowered.com/app/913740/WORLD_OF_HORROR/" target="_blank"><img class="ito-images" src="images/Ito2.jpg"/></a>
+    <a href="http://www.craigdavidson.net/nickcutter/index.html" target="_blank"><img class="ito-images" src="images/Ito3.jpg"/></a>
     </div>
-    <a href="templates/comments.html" target="_blank"><h1>Comment Section</h1></a>
+    </div>
+    <footer class="footer">
+      <a href="index.html"><p>About</p></a>
+      <p>&#x2609;</p>
+      <a href="templates/comments.html"><p>Contact</p></a>
```

Comment:
Any reason you took out the `target` attribute on the link

Reply:
I wanted the link to open in the current context, which is the default if no target is specified.

Reply:
There was not, I have added it back in